### DISCUSSION TITLE
MONAILABEL Slicer plugin | pelvis segmentation in MRI

### DISCUSSION THREAD

**Discussion by mahdiskj** [Project-MONAI/MONAILabel#673] [Q&A]
I want to segment pelvic in MRI from the SMIR dataset using the MONAILABEL plugin. I have read relatively a lot about this plugin. However, I can’t perform the segmentation well enough yet.

These are the steps I take to do so:
1.  connecting the server using the anaconda prompt
2.  enabling the plugin in Slicer and loading one image
3.  labeling the pelvis manually using the paint button in the scribbles part and the updating
4.  click “submit label”
and then repeating the process for some other images while the network is being trained.

_After the training process, I actually could not see anything when opening the mask files.
I also encounter these two errors:
" AssertionError: Not a valid Label "
" TypeError: object of type ‘NoneType’ has no len() "_

- Is it the correct way of using MONAI or am I missing something?
- Which aspects should I take into account before starting the process?

I would appreciate your help and suggestions.
Thank you

**Comment by diazandr3s** [comment]
Hi @mahdiskj, sorry I just saw this.
Can I ask you which MONAI Label app are you using (deepedit, deepgrow, or segmentation)?
Can you see the masks you’ve generated in the labels->final folder?
Can you also please share the log file (app.log) you have in the logs folder? It is easier for me to check what could happen.

Looking forward to hearing from you.
